lib/edit/style/SLDStyleConverter.js

```diff
@@ -98,6 +101,10 @@ exports.SLDStyleConverter = function() {
                         wellKnownName: style.symbol && style.symbol.shape || 'circle'
                     }];
             }
+            var opacity = 1;
+            if (style.symbol && angular.isDefined(style.symbol.fillOpacity)) {
+                opacity = Math.max(0.01, style.symbol.fillOpacity) / 100;
```

Comment:
@bartvde I guess it's worth looking at but I found opacity wasn't working until this was added